WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -263,6 +308,10 @@ struct AddEditCoupon: View {
             .navigationBarTitleDisplayMode(.large)
             .wooNavigationBarStyle()
         }
+        .navigationViewStyle(.stack)
+        .onDisappear {
+            onDisappear()
+        }
```

Comment:
@itsmeichigo I noticed after merging this branch with trunk that a `onDisappear` behavior was added to the `AddEditCoupon` creation, since this view ended up getting wrapped by the `UIHostingController`, the only solution I found to keep this behavior was to also connect the event through a closure callback. WDYT?

Reply:
This looks good to me 💯